tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -203,6 +205,272 @@ def test_brand_string(uvm_plain_any):
+                "cqm",
+                "cqm_llc",
+                "cqm_mbm_local",
+                "cqm_mbm_total",
+                "cqm_occup_llc",
+                "dca",
+                "ds_cpl",
+                "dtes64",
+                "dtherm",
+                "dts",
+                "epb",
+                "ept",
+                "ept_ad",
+                "est",
+                "flexpriority",
+                "flush_l1d",
+                "hwp",
+                "hwp_act_window",
+                "hwp_epp",
+                "hwp_pkg_req",
+                "ida",
+                "intel_ppin",
+                "intel_pt",
+                "mba",
+                "monitor",
+                "pbe",
+                "pdcm",
+                "pebs",
+                "pln",
+                "pts",
+                "rdt_a",
+                "sdbg",
+                "smx",
+                "tm",
+                "tm2",
+                "tpr_shadow",
+                "vmx",
+                "vnmi",
+                "vpid",
+                "xtpr",
+            }
+            assert guest_feats - host_feats == {
+                "hypervisor",
+                "tsc_known_freq",
+                "umip",
+            }
+        case CpuModel.INTEL_CASCADELAKE:
+            assert host_feats - guest_feats == {
+                "acpi",
+                "aperfmperf",
+                "arch_perfmon",
+                "art",
+                "bts",
+                "cat_l3",
+                "cdp_l3",
+                "cqm",
+                "cqm_llc",
+                "cqm_mbm_local",
+                "cqm_mbm_total",
+                "cqm_occup_llc",
+                "dca",
+                "ds_cpl",
+                "dtes64",
+                "dtherm",
+                "dts",
+                "epb",
+                "ept",
+                "ept_ad",
+                "est",
+                "flexpriority",
+                "flush_l1d",
+                "hwp",
+                "hwp_act_window",
+                "hwp_epp",
+                "hwp_pkg_req",
+                "ida",
+                "intel_ppin",
+                "intel_pt",
+                "mba",
+                "monitor",
+                "pbe",
+                "pdcm",
+                "pebs",
+                "pln",
+                "pts",
+                "rdt_a",
+                "sdbg",
+                "smx",
+                "tm",
+                "tm2",
+                "tpr_shadow",
+                "vmx",
+                "vnmi",
+                "vpid",
+                "xtpr",
+            }
+            assert guest_feats - host_feats == {
+                "hypervisor",
+                "tsc_known_freq",
+                "umip",
+            }
+        case CpuModel.INTEL_ICELAKE:
+            host_guest_diff_5_10 = {
+                "dtes64",
+                "hwp_act_window",
+                "pdcm",
+                "acpi",
+                "aperfmperf",
```

Comment:
SL/CL/IL have some commonalities. Can we extract those to make the diff smaller?

Reply:
I think this will only make it harder to maintain as features will be split into common groups spread across the file. Also optimizing for the diff size is not very useful.